configure.ac

```diff
@@ -50,6 +50,11 @@ RR_CHECK_FEATURE_LIB(TENSORFLOW, TensorFlow Installation,
                      tensorflow, TF_Version, tensorflow/c/c_api.h, no)
 
 AC_LANG_PUSH([C++])
+
+if test x$enable_edgetpu = xyes; then
```

Comment:
Howcome we are checking the variable first and then installing the feature? How does this work?

Reply:
Here I'm defining the shell variable enable_feature that macro [AC_ARG_ENABLE](https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/html_node/Package-Options.html) evaluates. Since we don't pass the --enable-onnxrt or --enable-tflite arguments we need to define those variables.